guide/src/README.md

```diff
@@ -1,13 +1,22 @@
 # Introduction
 
-> cargo, make me a project
+```rust
+cargo generate /me/a-project
+```
 
 [![Build status](https://github.com/cargo-generate/cargo-generate/workflows/Build/badge.svg)](https://github.com/cargo-generate/cargo-generate/actions?query=workflow%3ABuild+branch%3Amain+)
 [![crates.io](https://img.shields.io/crates/v/cargo-generate.svg)](https://crates.io/crates/cargo-generate)
 [![dependency status](https://deps.rs/repo/github/cargo-generate/cargo-generate/status.svg)](https://deps.rs/repo/github/cargo-generate/cargo-generate)
 
-`cargo-generate` is a developer tool to help you get up and running quickly with a new Rust
-project by leveraging a pre-existing git repository as a template.
+`cargo-generate` is a Rust developer tool to quickly setup any Rust project,
```

Comment:
> is a Rust developer tool

technically it is not a rust specific developer tool, it could be used in several rust unrelated contexts for example templates containing only Markdown or SQL. Hence I'd like to keep that part as open as possible.

Reply:
> technically it is not a rust specific developer tool, 

Hmm, don't you need Cargo to use it? 

OK so you want the PR to remove 'Rust' in the following:

> .. a new Rust project by ...

Reply:
what I'm trying to say is that the changed description does not speak to me so well. Do you really think that it would bring more clarity than it was before?

Reply:
what I'm trying to say is that the changed description does not speak to me so well. Do you really think that it would bring more clarity than it was before?